### PR TITLE
IA-3052 Bioconductor 3.15 and 3.14 - latest devel and release image rebuild

### PR DESCRIPTION
**Pull request by nturaga**
- Update to Bioconductor 3.15 - latest devel
- Rebuild Bioconductor devel image with 3.15 version

- Update to release image to Bioconductor 3.14 - latest release
- Rebuild Bioconductor devel image with 3.14 version

### CODE REVIEW

**Comment by nturaga** [comment]
@gpcarr @rtitle Any idea if this can be merged?

File: anvil-rstudio-bioconductor-devel/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 3.15.0 - 11/2/2021
+
+- Rebuild stack with latest devel image of Bioconductor with version 3.15
```
Comment (gpcarr) [inline anvil-rstudio-bioconductor-devel/CHANGELOG.md:3]: isn't the latest version of bioc 3.14?

Comment (nturaga) [reply anvil-rstudio-bioconductor-devel/CHANGELOG.md:3]: Yes, i'm updating two images here. 

latest development version is 3.15 

latest release version is 3.14

Comment (gpcarr) [reply anvil-rstudio-bioconductor-devel/CHANGELOG.md:3]: i don't see any changes to the development dockerfile in this PR - what does rebuilding the image as is update?

**Review by gpcarr — APPROVED** [review]

**Pull request merged**